src/apps/inventory/models.py

```diff
@@ -22,6 +32,14 @@ class Product(models.Model):
         limit_choices_to={"user_type": "Seller"},
     )
     created_at = models.DateTimeField(auto_now_add=True)
+    slug = models.SlugField(unique=True)
 
     def __str__(self):
         return self.name
+
+    def save(self, *args, **kwargs):
+        self.slug = slugify(self.name)
```

Comment:
be aware that if you ever update the name of a category or product, the slug won't automatically update. 
 Maybe we should consider using something like  https://pypi.org/project/django-autoslug/
@egorvavilov what do you think? I haven't use django for a while

Reply:
exactly, thank you for noticing!

Reply:
done